snuba/writer.py

```diff
@@ -95,3 +95,8 @@ def write(self, values: Iterable[bytes]) -> None:
                 / 1000
             )
         )
+
+
+class MockEncoder(Encoder[bytes, TDecoded]):
+    def encode(self, x: TDecoded) -> bytes:
+        return str(x).encode("utf-8")
```

Comment:
What is this for.

Reply:
Because I changed the signature of `InsertBatchWriter` to take a `BatchWriterEncoderWrapper[WriterTableRow]` instead of a `BatchWriter[JsonRow]` I needed to create a BatchWriterEncoderWrapper object in the `build_mock_batch_writer` method at https://github.com/getsentry/snuba/blob/8c5732968106e690db0d5cc39580be3a5dd7ae80/snuba/consumers/consumer.py#L325

Reply:
ohhhh that mock. Right, ignore my comment.